niworkflows/viz/utils.py

```diff
@@ -256,11 +303,17 @@ def compose_view(bg_svgs, fg_svgs, ref=0, out_file='report.svg'):
     import svgutils.transform as svgt
 
     # Read all svg files and get roots
-    svgs = [svgt.fromfile(f) for f in bg_svgs + fg_svgs]
+    svgs = [svgt.fromstring(f) for f in bg_svgs + fg_svgs]
     roots = [f.getroot() for f in svgs]
-    nsvgs = len(bg_svgs)
+
     # Query the size of each
-    sizes = np.array([(int(f.width[:-2]), int(f.height[:-2])) for f in svgs])
+    sizes = []
+    for f in svgs:
+        viewbox = f.root.get("viewBox").split(" ")
+        width = int(viewbox[2])
+        height = int(viewbox[3])
+        sizes.append((width, height))
+    nsvgs = len(bg_svgs)
```

Comment:
`sizes` is now expected to be an `ndarray`.

`sizes = np.array(sizes)` here should fix things.